Sources/SwiftFormatRules/ValidateDocumentationComments.swift

```diff
@@ -95,6 +97,22 @@ public final class ValidateDocumentationComments: SyntaxLintRule {
       diagnose(.documentReturnValue(funcName: name), on: returnClause)
     }
   }
+
+  /// Ensures the function has throws documentation if it may actually throw
+  /// an error.
+  private func validateThrows(
```

Comment:
It's probably worth an additional brief paragraph of documentation here to say that it's an explicit choice to **not** consider functions that use `rethrows` as needing `Throws` tags.

I wasn't sure if there was an established practice on this, but at least in the standard library, none of the functions that use `rethrows` document their errors; the only functions that have `Throws` tags are in `Result` and the various `Codable`-related types, which are all `throws`. So following that precedent (as you're already doing) makes sense.

Reply:
I was going off of the default behavior in Xcode, which in the release version is to crash 😜 and in the beta is to *not*  stub out a `- Throws` comment for functions that rethrow. 

But yeah, good call on calling that out explicitly.